documentation/asciidoc/stories/assembly_authorization.adoc

```diff
@@ -0,0 +1,17 @@
+ifdef::context[:parent-context: {context}]
+[id='authorization']
+:context: authorization
+= Configuring user roles and permissions
+
+[role="_abstract"]
+Secure access to {brandname} services by configuring role-based access control (RBAC) for users.
+Users must then have permissions that authorize access to caches and {brandname} resources.
```

Comment:
Yeah that wording is a little clunky now that I see it in the PR. Does this revision work for you? 

"This requires you to assign roles to users so that they have permission to access caches and {brandname} resources."

Reply:
Perfect!